docs/data/data-grid/export/export.md

```diff
@@ -36,7 +36,7 @@ By default, the export menu displays all the available export formats, according
 
 - [Print](#print-export)
 - [CSV](#csv-export)
-- [Excel](#excel-export) [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/) (🚧 Not delivered yet)
+- [Excel](#excel-export) [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
```

Comment:
We will need to update [this line](https://github.com/mui/material-ui/blob/424a31de18680baab7442c8cca8ff8c2530f8eb3/docs/src/components/pricing/PricingTable.tsx#L582)